javaparser-core/src/main/java/com/github/javaparser/JavaToken.java

```diff
@@ -335,14 +633,15 @@ public int hashCode() {
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-
+        if (this == o)
+            return true;
+        if (o == null || getClass() != o.getClass())
+            return false;
         JavaToken javaToken = (JavaToken) o;
-
-        if (kind != javaToken.kind) return false;
-        if (!text.equals(javaToken.text)) return false;
-
+        if (kind != javaToken.kind)
```

Comment:
I personally always use braces with the if statement

Reply:
Nothing happened here but reformatting of IDE generated code.